ceph/ceph_admin/host.py

```diff
@@ -105,6 +107,15 @@ def add(self, config):
         # Add host
         self.shell(args=cmd)
 
+        if _labels:
+            if "_admin" in _labels:
+                time.sleep(120)
```

Comment:
Can we have monitoring of file existence and non-existence instead of hard wait of 120 seconds.
```
def file_or_path_exists(node, file, timeout=120):
    
      while interval:
           ... validate and continue/break based on the existence or non-existence.
           ...
           sleep(5)    # wait time between intervals
           interval -= 1
      else:
           return False
      return true
```

Reply:
check if this above monitoring code changes is useful.

Reply:
I have implemented this and run is successful